sampleApp/sample.tsx

```diff
@@ -36,28 +46,39 @@ oneNoteDataProvider.getNotebooks().then((notebooks) => {
 			notebookListUpdater: updater,
 			callbacks: {
 				onNotebookHierarchyUpdated: (newNotebookHierarchy) => {
+					let latestNotebook = newNotebookHierarchy[newNotebookHierarchy.length - 1].name;
+					if(defaultDropdownLabel === "")
+						defaultDropdownLabel = latestNotebook; 
 					render(globalProps, newNotebookHierarchy);
+					renderDropdown(globalProps, newNotebookHierarchy, defaultDropdownLabel, true);
 				},
 				onSectionSelected: (section, breadcrumbs) => {
 					globalProps.globals.selectedId = section.id;
-
+					defaultDropdownLabel = section.name; 
 					// tslint:disable-next-line:no-console
 					console.log(breadcrumbs.map(x => x.name).join(' > '));
 
 					render(globalProps, globalProps.globals.notebookListUpdater!.get());
+					renderDropdown(globalProps, globalProps.globals.notebookListUpdater!.get(),  defaultDropdownLabel, false);
 				},
 				onPageSelected: (page, breadcrumbs) => {
 					globalProps.globals.selectedId = page.id;
-
+					defaultDropdownLabel = page.name;
 					// tslint:disable-next-line:no-console
 					console.log(breadcrumbs.map(x => x.name).join(' > '));
+					console.log(breadcrumbs[breadcrumbs.length -1].name)
 
 					render(globalProps, globalProps.globals.notebookListUpdater!.get());
+					renderDropdown(globalProps, globalProps.globals.notebookListUpdater!.get(), defaultDropdownLabel, false);
 				},
 				onAccessibleSelection: (selectedItemId: string) => {
 					globalProps.globals.ariaSelectedId = selectedItemId;
-
+					let notebookName = findNotebook(notebooks, selectedItemId);
+					if (defaultDropdownLabel === "")
```

Comment:
From a style perspective, we use single quotes, not double quotes. You can download TSLint extension for VSCode, which helps a lot with spotting lint errors.